modules/rostests/apitests/win32nt/ntuser/NtUserGetCursorInfo.c

```diff
@@ -0,0 +1,60 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for NtUserCreateAcceleratorTable
+ * COPYRIGHT:   Copyright 2025 Max Korostil (mrmks04@yandex.ru)
+ */
+
+#include "../win32nt.h"
+
+
+
+START_TEST(NtUserGetCursorInfo)
+{
+    CURSORINFO cursor;
+    DWORD error;
+    BOOL res;
+
+    // 1
```

Comment:
Maybe add a bit more informative comments for each case, about what are you currently testing, as in the following suggestion?
```suggestion
    /* 1. NULL pointer */
```